docs/_docs/installation.md

```diff
@@ -37,11 +46,34 @@ Before you start, make sure your system has the following:
   </p>
 </div>
 
+## Install & Use Jekyll with Bundler
+
+To use Jekyll with [Bundler](https://bundler.io/), add it as a
+dependency of your project. It can be installed in a sub-directory of the
+project. This avoids permissions issues, and also allows you to use different
+versions of Jekyll with different projects.
+
+First, install Bundler (as a system-wide gem):
+```sh
+sudo gem install bundler
```

Comment:
If the goal is to avoid permissions issues, using `sudo` is a non-starter.

Reply:
what is the recommended method then?

Reply:
`gem install --user-install bundler`

Reply:
Agreed. I've always installed Bundler along with Ruby via apt on Ubuntu, but if you're on LTS it's a really old version of Bundler... But you're right, we shouldn't recommend that.